File: OOP_C++_JAVA/lab3.cpp
#include <bits/stdc++.h>
using namespace std;
class Max_Min
{
public:
    int n1, n2;
    Max_Min(int n1, int n2)
    {
        this->n1 = n1;
        this->n2 = n2;
    }
    void print()
    {
        int result = max(n1, n2);
        cout << "Maximum = " << result << endl;
        int result_2 = min(n1, n2);
        cout << "Minimum = " << result_2 << endl;
    }
};
int main()
{
    int n1, n2;
    cout << "Enter the two numbers :";
    cin >> n1 >> n2;
    Max_Min obj = Max_Min(n1, n2);
    obj.print();

    return 0;
}

File: OOP_C++_JAVA/lab3.java
import java.util.Scanner;

class MaxMin {
   public int n1, n2;

  public MaxMin(int n1, int n2) {
        this.n1 = n1;
        this.n2 = n2;
    }

   public  void print() {
        int result = Math.max(n1, n2);
        System.out.println("Maximum = " + result);
        int result2 = Math.min(n1, n2);
        System.out.println("Minimum = " + result2);
    }
}

public class lab3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the two numbers: ");
        int n1 = scanner.nextInt();
        int n2 = scanner.nextInt();
        MaxMin obj = new MaxMin(n1, n2);
        obj.print();
        scanner.close();
    }
}
